website/docs/tutorials/podtato-head.md

```diff
@@ -0,0 +1,126 @@
+---
+id: podtato-head
+title: Let's start with Podtato-head
+sidebar_label: Podtato-head
+---
+
+---
```

Comment:
We can remove this and say ##Introduction

Reply:
I’ve updated the sidebar_label based on your suggestion. Please check the latest commit.